code/modules/ninja/suit/suit_attackby.dm

```diff
@@ -11,6 +11,15 @@
 			to_chat(U, "<span class='notice'>There are now [a_boost] adrenaline boosts remaining.</span>")
 			return
 
+
+	if(istype(I, /obj/item/reagent_containers/glass))//If it's a glass beaker.
```

Comment:
Duplicate of the previous if

Reply:
Ah, I see what you mean. If you would like, I can change it around to look less spaghetti-ish.